src/components/views/Article/Article.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Tags from 'views/Tags';
+import './Article.scss';
+
+class Article extends Component {
+    render() {
+        const { id, date, title, author, tags, content } = this.props.post,
+            pub_date = new Date(date * 1000);
+
+        return (
+            <article className="article-body">
+                <header>
+                    <div className="article-date">{pub_date.toDateString()}</div>
```

Comment:
1. I think that for this case you can define variable fully at the line 8 or modify it out of return method - it will allow you to debug it easilly in future
2. You can think about using special libraries for dates manipulations